rpc/rpcserver/server.go

```diff
@@ -117,21 +117,6 @@ func errorCode(err error) codes.Code {
 	}
 }
 
-// decodeAddress decodes an address and verifies it is intended for the active
-// network.  This should be used preferred to direct usage of
-// dcrutil.DecodeAddress, which does not perform the network check.
-func decodeAddress(a string, params *chaincfg.Params) (dcrutil.Address, error) {
```

Comment:
I'd like to keep this.  After decoding the address, we should still ensure that it is intended for the active network.  Only change is that the DecodeAddress call in the function body needs to be updated to remove the params.